projects/plugins/protect/src/js/components/ignore-threat-modal/index.jsx

```diff
@@ -0,0 +1,78 @@
+import { Button, getRedirectUrl, Text } from '@automattic/jetpack-components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { Icon } from '@wordpress/icons';
+import { STORE_ID } from '../../state/store';
+import styles from './styles.module.scss';
+
+const IgnoreThreatModal = ( { id, title, label, icon, severity } ) => {
+	const { setModal, ignoreThreat } = useDispatch( STORE_ID );
+	const threatsUpdating = useSelect( select => select( STORE_ID ).getThreatsUpdating() );
+	const codeableURL = getRedirectUrl( 'jetpack-protect-codeable-referral' );
+
+	const handleCancelClick = () => {
+		return event => {
+			event.preventDefault();
+			setModal( { type: null } );
+		};
+	};
+
+	const handleIgnoreClick = () => {
+		return async event => {
+			event.preventDefault();
+			ignoreThreat( id, () => {
+				setModal( { type: null } );
+			} );
+		};
+	};
+
+	return (
+		<>
+			<Text variant="title-medium" mb={ 2 }>
+				{ __( 'Do you really want to ignore this threat?', 'jetpack-protect' ) }
+			</Text>
+			<Text mb={ 3 }>{ __( 'Jetpack will ignore the threat:', 'jetpack-protect' ) }</Text>
+
+			<div className={ styles.threat }>
+				<Icon icon={ icon } className={ styles.threat__icon } />
+				<div className={ styles.threat__summary }>
+					<Text className={ styles.threat__summary__label } mb={ 1 }>
+						{ label }
+					</Text>
+					<Text className={ styles.threat__summary__title }>{ title }</Text>
+				</div>
+				<div className={ styles.threat__severity }>
+					{ /* to do: implement severity badge component once available */ }
+					{ severity }
+				</div>
+			</div>
+
+			<Text mb={ 4 }>
+				{ createInterpolateElement(
+					__(
+						'By ignoring this threat you confirm that your have reviewed the detected code and assume the risks of keeping a potentially malicious file on your site. If you are unsure please request an estimate with <codeableLink>Codeable</codeableLink>.',
```

Comment:
`your` -> `you`.

Beyond that, did this language come from the designs? Do we need to be quite so severe here? AFAIK we're not in Calypso, but maybe I missed that.

Reply:
Also, as your screenshot demonstrates, the `potentially malicious` part is not always accurate. If we don't want to tailor this based on the threat, we should do something like `potentially malicious or vulnerable`.

Reply:
Yep, I actually found that later in the day by accident when I was searching for Codeable stuff. It's fine to match it here.